instruqt-tracks/sentinel-for-terraform-v4/02-restrict-aws-iam-v1/assignment.md

```diff
@@ -0,0 +1,140 @@
+---
+slug: restrict-aws-iam-v1
+id: 5zp6wklyvaf2
```

Comment:
Let's clear the `id:` fields out of `assignment.md` while we're moving to v3 of the track config